src/sortStudents.ts

```diff
@@ -1,16 +1,71 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname ='surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averagegrades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc'|'desc';
 
+function getAverageGrades(grades:number[]):number {
+  return (grades
+    .reduce((sum:number, grade:number) => sum + grade) / grades.length);
+}
```

Comment:
If you use built-in reduce method in array of numbers - TS will know all types.

```suggestion
function getAverageGrades(grades:number[]):number {
  return grades.reduce((sum, grade) => sum + grade) / grades.length;
}
```

Reply:
Done.